clang/lib/Sema/JumpDiagnostics.cpp

```diff
@@ -18,6 +18,7 @@
 #include "clang/AST/StmtObjC.h"
 #include "clang/AST/StmtOpenACC.h"
 #include "clang/AST/StmtOpenMP.h"
+#include "clang/AST/StmtSYCL.h"
```

Comment:
Why is this here?

Reply:
Hmm, looks like no reason at all. I'm guessing it was once needed for some X-Macro usage prior to some rebase. I'll remove it.